src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age > 0 && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+
+    <p className="Person__partner">
+      {!person.isMarried && ('I am not married')}
+      {(person.isMarried && person.sex === 'f') && (`${person.partnerName} is my husband`)}
+      {(person.isMarried && person.sex === 'm') && (`${person.partnerName} is my wife`)}
```

Comment:
1. use ternary operator
2. create `maritalStatus` to avoid using this condition twice  
```
const maritalStatus = person.sex === 'f' ? : 'husband' : 'wife'
```

```suggestion
      {isMarried ? `${partnerName} is my ${maritalStatus}` : 'I am not married'}
```